logdna/logdna.py

```diff
@@ -238,6 +238,15 @@ def emit(self, record):
                     message['meta'][key] = record[key]
 
         message['meta'] = sanitize_meta(message['meta'], self.index_meta)
+
+        opts = {}
+        if 'args' in record and not isinstance(record['args'], tuple):
+            opts = record['args']
+
+        for key in ['app', 'env', 'hostname', 'level', 'timestamp']:
+            if key in opts:
+                message[key] = opts[key]
+
```

Comment:
Did these get removed before? Just curious what the root cause was.

Reply:
@teeohhem, while working on specifically meta fields, this section was removed and gone unnoticed